sympy/tensor/index_methods.py

```diff
@@ -24,6 +23,7 @@ class IndexConformanceException(Exception):
     pass
 
 
+@deprecated(useinstead='EinsteinSum', issue=9284)
 def _remove_repeated(inds):
```

Comment:
Methods that start with `_` are not public methods and do not, in general, need deprecation warnings.